apps/docs/docs/learn/02-thinking-in-stylex.mdx

```diff
@@ -249,15 +249,15 @@ type Props = {
 ```
 
 Styles being typed enables extremely sophisticated rules about the ways in which
-a component styles can be customized with **zero-runtime cost**.
+a component's styles can be customized with **zero-runtime cost**.
 
 ### Shareable constants
 
-CSS class names, CSS variables and other CSS identifiers are defined in a global
-namespace. The obvious and common API to bring them into JavaScript is as
+CSS class names, CSS variables, and other CSS identifiers are defined in a global
+namespace. The obvious and common API to bring them into JavaScript are as
 strings. However, this means losing type-safety and composability.
 
-We want styles to be type-safe and so we've spent a lot of time coming up with
+We want styles to be type-safe, so we've spent a lot of time coming up with
 APIs to replace these magic strings with references to Javacript constants. So
 far this is reflected in the following APIs:
```

Comment:
"StyleX is designed to be type-safe, so it provides APIs that avoid magic strings and instead uses references to JavaScript constants."

Note: "Javacript" is a typo